src/utils/loadScript.ts

```diff
@@ -24,10 +24,19 @@ const scriptExists = () => {
 };
 
 const appendScript = (): HTMLScriptElement => {
-  const { vaultId, environment, version } = getConfig();
+  const { vaultId, environment, version, integrity, crossorigin } = getConfig();
   const script = document.createElement('script');
 
   script.src = `${scriptURL}/vgs-collect/${version}/vgs-collect.js?sessionId=${SESSION_ID}&tenantId=${vaultId}&env=${environment}`;
+
+  if (integrity) {
+    script.integrity = integrity;
+  }
+
+  if (crossorigin) {
```

Comment:
according to MDN "" is a valid value which equals to anonymous, so maybe we need a more complex condition here
```suggestion
  if (typeof value === 'string') {
```